include/print.h

```diff
@@ -0,0 +1,43 @@
+#ifndef __CHERI_EXAMPLE_PRINT_H
+#define __CHERI_EXAMPLE_PRINT_H
+
+#include <assert.h>
+#include <cheri.h>
+#include <cheri/cheri.h>
+#include <cheri/cheric.h>
+#include <machine/sysarch.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/sysctl.h>
+#include <sys/types.h>
+
+//
+#define cheri_tag_get(x) __builtin_cheri_tag_get(x)
+#define cheri_address_get(x) __builtin_cheri_address_get(x)
+#define cheri_is_sealed(x) __builtin_cheri_sealed_get(x)
+#define cheri_flags_get(x) __builtin_cheri_flags_get(x)
+
+/*
+ * Print information about a capability
+ */
+
+void print_cap(void *__capability ptr)
```

Comment:
I think there's a way in `printf` to print out stuff about a capability, but I admit that I haven't tried it in hybrid!

Reply:
We have a `pp_cap` function for that in this repo [1]. Could reuse that if it doesn't complicate the building script.

[1] https://github.com/capablevms/cheri-examples/blob/master/include/common.h

Reply:
I know, but it includes cheriintrin.h, which I disabled for now, due to some conflicts in type definition.